### PR TITLE
repository2

### PR DESCRIPTION
**Pull request by quannb99**
## Related Tickets:
- [#31913](https://edu-redmine.sun-asterisk.vn/issues/31913)

## WHAT this PR does?
- Repository part 2

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by quannb99** [comment]
ready

File: app/Repositories/Like/LikeRepository.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        Like::updateOrCreate([
+            'user_id' => $userId,
+            'likeable_id' => $likeableId,
+            'likeable_type' => $likeableType,
+        ]);
+    }
+
+    public function deleteLike($userId, $likeableId, $likeableType)
+    {
+        $like = Like::where('user_id', $userId)->where('likeable_id', $likeableId)->where('likeable_type', $likeableType);
```
Comment (huyenttm-0901) [inline app/Repositories/Like/LikeRepository.php:26]: Nếu `$like = null` có bị lỗi delete ko em? em check chưa?

File: app/Repositories/User/UserRepository.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        }
+    }
+
+    public function deleteUser($id)
+    {
+        $user = User::findOrFail($id);
+        $user->delete();
+    }
+
+}
```
Comment (huyenttm-0901) [inline app/Repositories/User/UserRepository.php]: bỏ dòng thừa

**Comment by quannb99** [comment]
ready

**Pull request merged**